### PR TITLE
getElementWidth added

### LINKED ISSUE #76: APIs missed after v5 migration.
@AaronDavidNewman @mscuthbert @sschmidtu please add to this issue broken APIs that you would like to be recoverd.

### PR DESCRIPTION
**Pull request by rvilarl**
fix #76
static function to measure text/glyphs as requested in #76  

No visual differences.

### CODE REVIEW

File: src/element.ts [hunk 1]
```diff
@@ -616,6 +616,29 @@ export class Element {
...
+    let txtCanvas: HTMLCanvasElement | OffscreenCanvas | undefined = Element.#txtCanvas;
+    // Create the canvas element that will be used to measure text.
+    if (!txtCanvas) {
+      if (typeof document !== 'undefined') {
+        txtCanvas = document.createElement('canvas'); // Defaults to 300 x 150. See: https://www.w3.org/TR/2012/WD-html5-author-20120329/the-canvas-element.html#the-canvas-element
+      } else if (typeof OffscreenCanvas !== 'undefined') {
+        txtCanvas = new OffscreenCanvas(300, 150);
+      }
+      Element.#txtCanvas = txtCanvas;
+    }
```
Comment (ronyeh) [inline src/element.ts:630]: Is this duplicated from Element.measureText()?

If so, we should extract this into a separate method that initializes the Element.#txtCanvas if it does not exist yet.

Maybe `Element.getTextMeasurementCanvas()` and put it right after Element.setTextMeasurementCanvas()? If it is undefined, it creates a new one. If it is not undefined, it just returns the current one.

File: tests/key_clef_tests.ts [hunk 2]
```diff
@@ -22,18 +23,12 @@ const ClefKeySignatureTests = {
...
-function getWidth(code: string) {
-  const el = new Element();
-  el.setText(code);
-  return el.getWidth();
-}
-
 const fontWidths = () => {
-  const sharpWidth = getWidth('\ue262' /*accidentalSharp*/) + 1;
-  const flatWidth = getWidth('\ue260' /*accidentalFlat*/) + 1;
+  const sharpWidth = Element.getElementWidth(Glyphs.accidentalSharp) + 1;
```
Comment (ronyeh) [inline tests/key_clef_tests.ts:27]: There is already a `Element.getWidth()`. It is unclear how this is different (if I were a developer new to VexFlow).

Should this be called `Element.measureWidth()`?

File: src/element.ts (hunk 1 above)
Comment (rvilarl) [reply src/element.ts]: done

File: tests/key_clef_tests.ts (hunk 2 above)
Comment (rvilarl) [reply tests/key_clef_tests.ts]: done

**Pull request merged**